AutomatedTesting/Gem/PythonTests/Atom/tests/hydra_AtomEditorComponents_LightAdded.py

```diff
@@ -97,74 +182,299 @@ def AtomEditorComponents_Light_AddedToEntity():
+                Report.result(test_light_shape,
+                              light_entity.has_component(light_shape))
 
-        # 8. Test IsVisible.
-        light_entity.set_visibility_state(True)
-        general.idle_wait_frames(1)
-        Report.result(Tests.is_visible, light_entity.is_visible() is True)
+            # 8. Edit Color parameter.
+            color_value = math.Color(1.0, 0.0, 0.0, 1.0)
+            light_component.set_component_property_value(AtomComponentProperties.light('Color'), color_value)
+            light_color = light_component.get_component_property_value(
+               AtomComponentProperties.light('Color'))
+            general.idle_wait_frames(1)
+            Report.result(Tests.edit_light_color, light_color.IsClose(color_value))
+
+            # 9. Set Intensity mode.
+            # if current_light_type < 6:
+            #     for intensity_mode in INTENSITY_MODE.keys():
+            #         light_component.set_component_property_value(
+            #             AtomComponentProperties.light('Intensity mode'), INTENSITY_MODE[intensity_mode])
+            #         current_intensity_mode = light_component.get_component_property_value(
+            #             AtomComponentProperties.light('Intensity mode'))
+            #         test_intensity_mode = (
+            #             f"Intensity mode set to {current_intensity_mode}",
+            #             f"P1: Intensity mode failed to be set to {current_intensity_mode}")
+            #         Report.result(test_intensity_mode,
+            #                       current_intensity_mode == INTENSITY_MODE[intensity_mode])
+
+            # 10. Edit the Intensity parameter.
+            light_component.set_component_property_value(AtomComponentProperties.light('Intensity'), 1000)
+            Report.result(Tests.edit_intensity_value,
+                          light_component.get_component_property_value(
+                              AtomComponentProperties.light('Intensity')) == 1000)
+
+            # 11. Set the Attenuation radius Mode:
+            for radius_mode in ATTENUATION_RADIUS_MODE.keys():
+                light_component.set_component_property_value(
+                    AtomComponentProperties.light('Attenuation radius Mode'), ATTENUATION_RADIUS_MODE[radius_mode])
+                current_radius_mode = light_component.get_component_property_value(
+                   AtomComponentProperties.light('Attenuation radius Mode'))
+                test_attenuation_mode = (
+                    f"Attenuation radius Mode set to {radius_mode}",
+                    f"P1: Attenuation radius Mode failed to be set to {radius_mode}")
+                general.idle_wait_frames(1)
+                Report.result(test_attenuation_mode,
+                              current_radius_mode == ATTENUATION_RADIUS_MODE[radius_mode])
+
+                # 12. Edit the Attenuation radius Radius parameter.
+                if current_radius_mode == 0:
```

Comment:
it's less clear when you branch off get values. instead use something more human readable
if radius_mode = ATTENUATION_RADIUS_MODE['explicit']:
also mention that step 12 is only executed if radius mode is 'explicit'

Reply:
changed to radius_mode == 'explicit' and noted in step 11 that this is explicit only.